app/Services/Themes/QueryThemesService.php

```diff
@@ -52,14 +52,16 @@ private static function applyBrowse(Builder $query, string $browse): void
     /** @param Builder<Theme> $query */
     private static function applySearch(Builder $query, string $search): void
     {
-        $query->where('name', 'like', "%$search%")
+        $query
+            ->whereFullText('slug', $search)
+            ->orWhereFullText('name', $search)
```

Comment:
If you look at the migration and compare this with the users table and plugins table, you'll find that there's nothing of note and you gained nothing from reading this comment, but do imagine me having a good chuckle to myself for being witty.

That's what you get for me waiting on the kettle boiling while I review the PR.